michaelwlin/s_capstone_ai_jobs
# Add dummy user data for later testing

## Description 
-Used faker module to generate user test data for later use
-`npm install faker`
- uses jest for testing

## To Test
Ensure you're in s_capstone_ai_jobs/server
Run `npm test`

Should see:
![image](https://github.com/michaelwlin/s_capstone_ai_jobs/assets/107959606/36f9474d-be4d-43c9-84a1-5213af68cc6b)
